recipe-server/client/actions/tests/test_preference-experiment.js

```diff
@@ -0,0 +1,136 @@
+import PreferenceExperimentAction from '../preference-experiment/';
+import { recipeFactory } from '../../tests/utils.js';
+
+function preferenceExperimentFactory({ ...args }) {
```

Comment:
Nothing. I had it as `(...args)` at first, realized that didn't work like `**kwargs`, and slapped the `{}` on without thinking too hard. I'll fix this.